### PR TITLE
Add testing for ParseAddProperty

### LINKED ISSUE #251: No testing for ParseAddProperty

### PR DESCRIPTION
**Pull request by OVReader**
Fix #251

### CODE REVIEW

File: src/test/java/seedu/duke/parsermanager/ParseAddPropertyTest.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import seedu.duke.exception.InvalidPriceFormatException;
+import seedu.duke.exception.InvalidSingaporeAddressException;
+import seedu.duke.exception.InvalidUnitTypeLabelException;
+import seedu.duke.exception.MissingAddPropertyFlagException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ParseAddPropertyTest {
+    // Valid Add Property Test Cases with varying Address Formats
```
Comment (wilsonngja) [inline src/test/java/seedu/duke/parsermanager/ParseAddPropertyTest.java:26]: The name of the constant should be quite intuitive already, thus perhaps there isn't a need for this line of comment anymore?

File: src/test/java/seedu/duke/parsermanager/ParseAddPropertyTest.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                LANDED_ADDRESS_WITH_STREET_NUMBER, LP_UNIT_TYPE);
+        parseCommand_validAddPropertyInput_matchingPropertyDetails(VALID_BUILDING_PROPERTY_WITH_STREET_NUMBER_INPUT,
+                BUILDING_ADDRESS_WITH_STREET_NUMBER, BUILDING_UNIT_TYPE);
+        parseCommand_validAddPropertyInput_matchingPropertyDetails(
+                VALID_BUILDING_PROPERTY_WITH_STREET_NAME_AND_BUILDING_NAME_INPUT,
+                BUILDING_ADDRESS_WITH_STREET_NUMBER_AND_BUILDING_NAME,
+                BUILDING_UNIT_TYPE);
+    }
+
+    private static void parseCommand_validAddPropertyInput_matchingPropertyDetails(String validInput,
```
Comment (wilsonngja) [inline src/test/java/seedu/duke/parsermanager/ParseAddPropertyTest.java:84]: Good job in applying KISS. It really helps alot when reading the code.

**Review by wilsonngja — COMMENTED** [review]
The coverage of ParseAddProperty is extensive. I really like the effort you have put in. The code LGTM :D

**Review by ngdeqi — COMMENTED** [review]
Follows code quality guidelines. LGTM

**Review by ngdeqi — APPROVED** [review]
LGTM

**Pull request merged**